internal/node/node.go

```diff
@@ -124,6 +124,7 @@ func NewBus(cfg BusConfig, dir string, seed types.PrivateKey, l *zap.Logger) (ht
 		GormLogger:                    sqlLogger,
 		SlabPruningInterval:           cfg.SlabPruningInterval,
 		SlabPruningCooldown:           cfg.SlabPruningCooldown,
+		RetryTransactionIntervals:     []time.Duration{200 * time.Millisecond, 500 * time.Millisecond, time.Second, 3 * time.Second, 10 * time.Second, 10 * time.Second},
```

Comment:
This seems like a bit of an odd thing to make configurable. Is this really necessary?

Reply:
No, not necessary at all, I agree it looks odd but it might be useful to some and I added it to avoid unnecessarily long retry intervals in testing. I had a test that took more than a minute to fail only because the tx was retried.

I don't mind reverting it but I do feel we don't need ~30s retries in testing (?)

Reply:
Ok that's fine but make sure to also add any errors you might encounter to the exception list for the retries. I wouldn't expect tests to retry more than once every full moon unless there is an error that we can't recover from in the first place.

Reply:
Ok will do! 👍